internal/export/model/export_model.go

```diff
@@ -58,8 +58,8 @@ func (ec *ExportConfig) Validate() error {
 	if ec.Period > oneDay {
 		return errors.New("maximum period is 24h")
 	}
-	if ec.Period == 0 {
-		return errors.New("period must be non-zero")
+	if ec.Period == 0 || ec.Period.Seconds() == 0 {
```

Comment:
if !(ec.Period > 0) ?

Reply:
The period could be 5 nanoseconds. That's > 0, but not > 1 second.

Reply:
if !(ec.Period.Seconds() >= 1)